ray-operator/controllers/ray/common/pod.go

```diff
@@ -755,12 +757,7 @@ func makeEmptyDirVolume(container *v1.Container, volumeName string, storageMediu
 func checkIfVolumeMounted(container *v1.Container, pod *v1.Pod, volumeMountPath string) bool {
 	for _, mountedVol := range container.VolumeMounts {
 		if mountedVol.MountPath == volumeMountPath {
-			for _, podVolume := range pod.Spec.Volumes {
```

Comment:
@DmitriGekhtman Dima, what is the original reason for adding this pod.Spec.Volumes comparison? I discussed with @scarlet25151 about this and we are not sure whether we need this part.

Reply:
It is the first commit. And you are the one that made changes to code at the closest place.... @DmitriGekhtman 

![image](https://user-images.githubusercontent.com/3913185/200720378-f7bc5186-0aba-44c5-96b6-b0cd23b2320d.png)

Reply:
Oh, I just reused the existing code to add a new volume mount for autoscaler logs.

Reply:
I don't think we need the extra check. Indeed, it's incorrect, as it allows redundant mountPaths!
Thanks for fixing it!

Reply:
thanks for the info

Reply:
FYI: @scarlet25151